errors/errors.js

```diff
@@ -0,0 +1,16 @@
+exports.handleCustomErrors = (err, req, res, next) => {
+    if (err.status && err.message) {
+        res.status(err.status).send({ message: err.message });
+    } else next(err);
+};
+
+exports.handlePsqlErrors = (err, req, res, next) => {
+   // console.log(err);
+    if (err.code = '22P02') res.status(400).send({ message: `Bad Request` })
```

Comment:
Great use of the error codes back from PSQL to send a meaningful error back to the user